recipes/bdwgc/all/conandata.yml

```diff
@@ -2,7 +2,13 @@ sources:
   "8.0.4":
     url: "https://github.com/ivmai/bdwgc/releases/download/v8.0.4/gc-8.0.4.tar.gz"
     sha256: "436a0ddc67b1ac0b0405b61a9675bca9e075c8156f4debd1d06f3a56c7cd289d"
+  "8.0.6":
+    url: "https://github.com/ivmai/bdwgc/releases/download/v8.0.6/gc-8.0.6.tar.gz"
+    sha256: "3b4914abc9fa76593596773e4da671d7ed4d5390e3d46fbf2e5f155e121bea11"
 patches:
   "8.0.4":
     - patch_file: "patches/update-cmake.patch"
       base_path: "source_subfolder"
+  "8.0.6":
+    - patch_file: "patches/update-cmake-8_0_6.patch"
```

Comment:
Is it possible to re-use the 8.0.4 patch, and apply patches on top of that?
Or did too many things change?

Reply:
Yes, it is possible to make a patch downgrading CMakeLists.txt to 8.0.4 and then apply existing update-cmake.patch. Worth doing?

Reply:
Only if the diffs are small.
it looks like the hooks are not complaining about the recipe size, so the current state is okay for me.

Reply:
OK, leaving it as is. At least having independent patches looks to me more natural.